WordPress/Classes/Services/CommentServiceRemoteCoreRESTAPI.swift

```diff
@@ -0,0 +1,323 @@
+import Foundation
+import WordPressCore
+import WordPressAPI
+import WordPressAPIInternal
+import WordPressKit
+import WordPressShared
+import WordPressData
+
+final class CommentServiceRemoteCoreRESTAPI: NSObject, CommentServiceRemote {
+    private let client: WordPressClient
+
+    init(client: WordPressClient) {
+        self.client = client
+    }
+
+    func getCommentsWithMaximumCount(_ maximumComments: Int, success: (([Any]?) -> Void)?, failure: (((any Error)?) -> Void)?) {
+        getCommentsWithMaximumCount(maximumComments, options: nil, success: success, failure: failure)
+    }
+
+    func getCommentsWithMaximumCount(_ maximumComments: Int, options: [AnyHashable: Any]? = [:], success: (([Any]?) -> Void)?, failure: (((any Error)?) -> Void)?) {
+        Task { @MainActor in
```

Comment:
Do we need this task to run on the `@MainActor` because of its dependencies?

IMHO ideally we'd do all of the processing off the main thread, but maybe that's just not possible yet?

Reply:
At some point in the future, we'll move the `Task` up to the UI layer, at which point this implementation will be deleted anyway.